settings/common.py

```diff
@@ -348,7 +350,7 @@
     "ALGORITHM": "HS256",
     "AUDIENCE": None,
     "ISSUER": None,
-    "AUTH_HEADER_TYPES": ("Bearer",),
+    "AUTH_HEADER_TYPES": ("Token",),
```

Comment:
@ShauryaAg we are not switching to `AccessToken`. We are using `RefreshToken` because we want to support token blacklisting. Can you revert this change.

Reply:
@Ram81 I did not change `RefreshToken` to `AccessToken`, it was only something I suggest as I was curious.

However, changing `Bearer` to `Token` is still needed, as `rest_auth` verifies and returns using request headers with `Authorization: Token <Token>` as set in the frontend.
https://github.com/Cloud-CV/EvalAI/blob/adde7be63e25af834d65ef2625103040122f561e/frontend_v2/src/app/services/api.service.ts#L26

Here, we are setting default headers for `Authorization` as `Token <Token>`. If we don't change `Bearer` to `Token` in `AUTH_HEADER_TYPES`, then we will need to set `Authorization` header as `Bearer <Token>` in the frontend.

Reply:
We don't want to switch to Bearer token on frontend side. To show the `RefreshToken` on the submit page just make an API call to `get_auth_token` API and use the response to show the token on UI. No need to change the authorization headers on frontend local storage